src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  margin: auto;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 7px;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-black);
+  margin-top: 40px;
+  width: 100%;
+}
+
+.card__text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__text--code {
+  color: var(--color-grey);
+  margin-top: 4px;
+  align-self: flex-start;
+}
+
+.card__text--review {
+  color: var(--color-black);
+}
+
+.card__container {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__container--rate {
+  margin-top: 16px;
+}
+
+.card__container--price {
+  margin-top: 24px;
+}
+
+.card__text--price {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-grey);
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-black);
+  text-align: right;
```

Comment:
it's redundant

Reply:
when styles were deleted, test has not passed

Reply:
You don't need to use `text-align`. You have flexbox here, so your blocks will align on the left and right side.